iree/compiler/Conversion/LinalgToLLVMGPU/Passes.h

```diff
@@ -28,6 +28,14 @@ createTileAndDistributeToThreads();
 
 std::unique_ptr<OperationPass<FuncOp>> createRemoveSingleIterationLoopPass();
 
+//
+std::unique_ptr<OperationPass<IREE::HAL::ExecutableTargetOp>>
+createLowerExecutableTargetGPUPass();
+
+void addGPUVectorizationPassPipeline(OpPassManager &passManager);
```

Comment:
Could you comment and what the difference between the two passes pipelines is?

Reply:
Added some comment, note that for now this is mostly there for the example I haven't really added interesting cases for dynamic pipeline.